src/libproxy/libproxy.map

```diff
@@ -1,3 +1,9 @@
+LIBPROXY_0.5.5 {
+  global:
+    px_proxy_factory_get_type;
+    px_proxy_factory_copy;
+};
```

Comment:
should be `} LIBPROXY_0.4.16;`

Reply:
See for the format of the map: https://sourceware.org/binutils/docs/ld/VERSION.html